assets/meta.yaml

```diff
@@ -410,3 +410,190 @@
+    explanation: See [[SA-1B website]](https://ai.facebook.com/datasets/segment-anything/)
+  prohibited_uses:
+    value: >
+      Authors note the following limitations of the dataset:
+        The masks are generated by a segmentation model, so there may be errors
+      or inconsistencies in the masks.
+        While no two images are the same, there are instances of images of the same
+      subject taken close together in time.
+        The dataset contains scenes of protests, or other gatherings that may suggest
+      religious beliefs, political opinions or union memberships that may be offensive.
+
+    explanation: See [[Datasheet]](https://arxiv.org/pdf/2304.02643.pdf#page=25)
+  monitoring:
+    value: >
+      The dataset will be hosted at https://ai.facebook.com/datasets/segment-anything
+      and maintained by Meta AI.
+      "If a user observes objectionable image(s) in the dataset, we invite them
+      to report the image(s) at segment-anything at meta.com for removal"
+      "To aid reproducibility of research using SA-1B, the only updates (to the
+      dataset) will be to remove reported images."
+      "We encourage users to gather further annotations for SA-1B. Any users who
+      generate annotations will be liable for hosting and distributing their annotations."
+    explanation: See [[Datasheet]](https://arxiv.org/pdf/2304.02643.pdf#page=25)
+  feedback: Feedback can be given via the feedback form on their website [segment-anything.com](https://segment-anything.com/)
+    or by emailing at segment-anything at meta.com.
+
+- type: model
+  name: SAM
+  # General
+  organization: Meta
+  description: A foundation model for image segmentation.
+  created_date:
+    value: 2023-04-05
+    explanation: The date the [[Meta blog post]](https://ai.facebook.com/blog/segment-anything-foundation-model-image-segmentation/)
+      was released.
+  url: https://arxiv.org/pdf/2304.02643.pdf
+  model_card:
+    value: https://arxiv.org/pdf/2304.02643.pdf#page=28
+    explanation: Model card can be found in the Appendix section of the paper.
+  modality: image, segmentation prompt (point, box, mask and text)
+  analysis:
+    value: >
+      "We extensively evaluate SAM.
+      First, using a diverse new suite of 23 segmentation datasets, we find that
+      SAM produces high-quality masks from a single foreground point, often only
+      slightly below that of the manually annotated ground truth.
+      Second, we find consistently strong quantitative and qualitative results on
+      a variety of downstream tasks under a zero-shot transfer protocol using prompt
+      engineering, including edge detection, object proposal generation, instance
+      segmentation, and a preliminary exploration of text-to-mask prediction."
+    explanation: See [[Zero-Shot Transfer Experiments]](https://arxiv.org/pdf/2304.02643.pdf#section.7)
+      for more details.
+  size: unknown
+  # Construction
+  dependencies: [SA-1B]
```

Comment:
In the paper they mention "Motivated by scalability and access to strong pre-training, we use an MAE [47] pre-trained Vision Transformer (ViT) [33] with minimal adaptations to process high resolution inputs, specifically a ViT-H/16 with 14×14 windowed attention and four equally-spaced global attention blocks, following [62]". They also use ViT-B/L, but "Unless otherwise specified: (1) SAM uses an MAE [47] pre-trained ViT-H [33] image encoder"

should ViT-H be added as a dependency?

Reply:
Good question, I think we should say ViT-H if it is a standard version of ViT-H, but if the pretrained it themselves with their own data, then no. Will look at the paper again

Reply:
From what I understand, standard version of ViT-H would be the one presented in the original [ViT paper](https://paperswithcode.com/paper/an-image-is-worth-16x16-words-transformers-1), but here they have used a ViT-H model that's pre-trained using the Masked Autoencoder approach from [this paper](https://arxiv.org/abs/2111.06377)